plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastoreClient.java

```diff
@@ -731,14 +753,14 @@ public void createFunction(Function function)
     public void alterFunction(Function function)
             throws TException
     {
```

Comment:
createFunction is still missing

Reply:
@anusudarsan createFunction does not allow to pass the dbName.
are you suggesting like this?

```
        function.setDbName(prependCatalogToDbName(catalogName, function.getDbName()));
        client.createFunction(function);
```

Reply:
updated